lib/src/client.dart

```diff
@@ -1158,12 +1160,213 @@ class Client extends MatrixApi {
     _archivedRooms.add(ArchivedRoom(room: archivedRoom, timeline: timeline));
   }
 
+  GetVersionsResponse? _versionsCache;
+
+  Future<bool> authenticatedMediaSupported() async {
+    _versionsCache ??= await getVersions();
+    return _versionsCache?.versions.contains('v1.11') ?? false;
```

Comment:
Actually it is supported if a version of "v1.11" or higher is supported, right? So what if a server claims to support v1.12 but not v1.11, could this happen?

Reply:
Hm I do not see the spec mentioning anything against this so it is possible yes. Nevertheless, In practice I don't think it's something we need to worry about. Fixing this ofc would mean some simple regex and checks, do we want to go that route?